modules/cpmso-ts-ilb-missing.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-ts-ilb-missing_{context}"]
+= Adding a missing Azure internal load balancer
+
+The `internalLoadBalancer` parameter is required in both the `ControlPlaneMachineSet` and control plane `Machine` custom resources (CRs) for Azure. If this parameter is not preconfigured on your cluster, you must add it to both CRs.
+
+For more information on where this parameter is located in the Azure provider specification, see the sample Azure provider specification. The placement in the control plane `Machine` CR is similar.
+
+.Procedure
+
+//Do they actually have to do this or do they just need to stick it in the CPMS CR and it will propagate from there?
+. List the control plane machines in your cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get machines -l machine.openshift.io/cluster-api-machine-role==master -n openshift-machine-api
+----
+
+. For each control plane machine, edit the CR by running the following command:
++
+[source,terminal]
+----
+$ oc edit machine <control_plane_machine_name>
+----
+
+. Add the `internalLoadBalancer` parameter with the correct details for your cluster and save your changes.
+
+//Re: previous note, do they just start here?
```

Comment:
This note is causing the numbering in the preview to restart, should be resolved when I remove it.

Reply:
IIRC they can't activate the CPMS without a valid value for this field, in which case, they will probably want to make sure the existing Machines are correct too

Reply:
Ok cool - I was thinking the CPMS would revert the changes to the machines themselves if they edited the machines first while the CPMS still had the old value, but seems that's not a valid concern